From 9bdbbb6cee4afa43b84a2055a02bd329aea805c4 Mon Sep 17 00:00:00 2001
From: Subrata Chatterjee <schatterjee@habana.ai>
Date: Fri, 24 Jun 2022 15:45:28 +0300
Subject: [PATCH] [SW-91832] gaudi2: erom: add sha file for recovery compilation

Currently, sha_csum_regions is empty in erom recovery code due to
which image verification fails as hash is not computed on fit data
regions while signature verification. This patch adds sha_csum_regions
function implemented in sha.c file to be part or erom recovery
compilation.

Change-Id: I588706060b62c0a6fb797826d28933b5e1b415e8
---

diff --git a/app/ext_rom/Makefile b/app/ext_rom/Makefile
index 7b1a31a..263d408 100644
--- a/app/ext_rom/Makefile
+++ b/app/ext_rom/Makefile
@@ -94,6 +94,7 @@
 	$(HL_DEV_ARCH)/../common/common.o \
 	$(HL_DEV_ARCH)/../common/security.o \
 	$(HL_DEV_ARCH)/../common/pll.o \
+	$(HL_DEV_ARCH)/../common/sha.o \
 	$(HL_DEV_ARCH)/../common/$(HL_TARGET).o \
 	$(HL_DEV_ARCH)/$(HL_TARGET).o
 
